.gitlab-ci/template/include/preferred-wpe-2_32.conf

```diff
@@ -1,6 +1,6 @@
 PREFERRED_PROVIDER_virtual/wpebackend = "wpebackend-fdo"
 PREFERRED_PROVIDER_virtual/libwpe = "libwpe"
-PREFERRED_VERSION_cog = "0.8.%"
-PREFERRED_VERSION_wpewebkit = "2.32.0%"
+PREFERRED_VERSION_cog = "0.10.%"
+PREFERRED_VERSION_wpewebkit = "2.32.1%"
```

Comment:
Wouldn't this be  `2.32.%`? With the current value of `2.32.1%` I understand that version 2.32.10 would be preferred (it would match the pattern) over 2.32.20 (which does not match).

Reply:
Fixed